.github/workflows/cron_nightly.yml

```diff
@@ -0,0 +1,66 @@
+name: Nightly
+
+on:
+  pull_request:
+    branches:
+      - issue22650
+  schedule:
+    - cron:  '30 2 * * *'
+
+jobs:
+  nightly:
+    runs-on: ubuntu-latest
+    name: Nightly
+    strategy:
+      matrix:
+        php: [ '7.3' ]
+    env:
+      GC_INSTANCE_KEY: ${{ secrets.GC_INSTANCE_KEY }}
+      GC_INSTANCE_NAME: ${{ secrets.GC_INSTANCE_NAME }}
+      #GC_INSTANCE_PUB: ${{ secrets.GC_INSTANCE_PUB }}
```

Comment:
forgotten?

Reply:
Used before but not actually so I prefer keeping and removing it when we are sure the cron runs well.